src/tests/databases/core_9/meta.txt

```diff
@@ -1,4 +1,4 @@
-12	1	assembly.accession	GCA_000002985.3
+12	1	assembly.accession	test1
```

Comment:
Why changing these values?

Reply:
This is the interdependent issue. On a previous update you swapped out the values for real ones and commented out the tests. These are changed as the tests were written to see if a previously created dataset is overridden. A rewrite is possible but not trivial for these to get proper values and independence incorporated.

Reply:
So the test that has been reactivated should either be inside the test on which it depends. or commented out again. These dependencies cause more issues, and are very error prone.